src/webui/abstractwebapplication.cpp

```diff
@@ -366,9 +367,9 @@ bool AbstractWebApplication::sessionStart()
 bool AbstractWebApplication::sessionEnd()
 {
     if ((session_ != 0) && (sessions_.contains(session_->id))) {
-        QNetworkCookie cookie(C_SID, session_->id.toUtf8());
```

Comment:
Is cookies are not identified by `name=value` pair? Is name only enough to remove cookie?